### PR TITLE
New tooling

### PR DESCRIPTION
**Pull request by galvesribeiro**
- Updated the tooling so people using VS2017 and latest released dotnet CLI can build the project again.
- Updated build scripts to match new tooling.
- Removed .nuspec files. They aren't necessary anymore with the new tooling.

### CODE REVIEW

**Comment by galvesribeiro** [comment]
@jterry75 Hey, I've updated to the new tooling here but for whatever reason AppVeyor is not able to clone the code from GH. Can you give a hand here? I'm trying to move forward with the rest of the issues to I can use it.

**Comment by jterry75** [comment]
I dont quite understand that failure from AppVeyor. Can you rebase and push an update to get AV to try again. Seems like a node failure. We dont control that part of the clone process.

**Comment by galvesribeiro** [comment]
@jterry75 same issue... I'll open another PR from another branch and see if it happen there...

**Comment by galvesribeiro** [comment]
Ok, it doesn't work @jterry75. Can you open a ticket on AV?

Thank you.

**Comment by jterry75** [comment]
Oh I didnt look closely enough. This is a bug in your AppVeyor.yml You cant copy to $PWD because that is the folder where the git checkout takes place (hence it is no longer empty). You need to copy to c: like we used to for staging or somewhere else.

**Comment by galvesribeiro** [comment]
@jterry75 ok, fixed :) 

Once we have it merged I'll look at the AutoRest thing. Thanks!

**Comment by jterry75** [comment]
Ok one more change. Can we keep passing the version number so the AppVeyor build increments like it used to. We can hard code the `major.minor` in the file and use `.*` to have appVeyor replace it

File: appveyor.yml [hunk 1]
```diff
@@ -2,14 +2,9 @@ version: '2.124.2.{build}'
...
-- ps: dotnet build .\src\Docker.DotNet\Docker.DotNet.csproj -c Release
-- ps: dotnet build .\src\Docker.DotNet.X509\Docker.DotNet.X509.csproj -c Release
-- ps: dotnet build .\src\Docker.DotNet.BasicAuth\Docker.DotNet.BasicAuth.csproj -c Release
-- ps: dotnet build .\test\Docker.DotNet.Tests\Docker.DotNet.Tests.csproj -c Release
-- ps: nuget pack Docker.DotNet.nuspec -properties version=$($env:appveyor_build_version)
-- ps: nuget pack Docker.DotNet.BasicAuth.nuspec -properties version=$($env:appveyor_build_version)
-- ps: nuget pack Docker.DotNet.X509.nuspec -properties version=$($env:appveyor_build_version)
\ No newline at end of file
+- ps: dotnet build -c Release
+- ps: dotnet pack -c Release -o $pwd /p:version=$(appveyor_build_version)
```
Comment (jterry75) [inline appveyor.yml]: `$($env:appveyor_build_version)`

Comment (galvesribeiro) [inline appveyor.yml]: Oh! right

**Pull request merged**